src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js

```diff
@@ -322,9 +322,11 @@ const OncoGridWrapper = compose(
         performanceTracker.begin('oncogrid:render');
         grid.render();
 
-        grid.on('render:all:end', () =>
-          performanceTracker.end('oncogrid:render', performanceContext),
-        );
+        grid.container[0][0].style.visibility = 'hidden';
+        grid.on('render:all:end', () => {
+          grid.container[0][0].style.visibility = 'visible';
```

Comment:
`grid.container` should be a d3 object, so you could get the node with `grid.container.node()`